test/promql/tests/test_unschedulable_pods.yaml

```diff
@@ -0,0 +1,45 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  # Interval for the samples of the series
```

Comment:
There some more generic comments throughout the file. Let's remove them as well.